app/controllers/posts_controller.rb

```diff
@@ -5,6 +5,29 @@ def index
   end
 
   def show
-    @post = Post.find(params[:id])
+    @current_user = current_user
+    @post = Post.find(params[:post_id])
+  end
+
+  def new
+    @current_user = current_user
+    @post = @current_user.posts.new
+  end
+
+  def create
+    @current_user = current_user
+    @post = @current_user.posts.new(post_params)
+
+    if @post.save
+      redirect_to user_posts_path(@current_user), notice: 'Post created successfully.'
+    else
+      render :new
+    end
```

Comment:
- [ ] Hello @banzaman, great effort working on your project, but I realized the form to add more posts to a particular user has not been achieved in your project. Kindly check the video recording below to have more insight on what I am talking about.

Video 👇 

[screen-capture (1).webm](https://github.com/banzaman/Blog-app/assets/78634841/4f1628e9-728f-489f-8c7c-c70874ae8209)

Reply:
I will work on that